backend/School.js

```diff
@@ -0,0 +1,83 @@
+const Class = require('./Class');
+const Student = require('./Student')
+
+class School {
+  constructor() {
+    this.classes = {
+      // className: Class Object
+        Physics: {
+          name: "Physics",
+          teacher: 'Neil de Grase Tyson',
+          students: ["Phil", "Fred"],
+        } 
+    }
+  }
+
+  /**
+   * Add class to classes
+   * 
+   * @param {string} name - Name of the class
+   * @param {string} teacher - Name of instructor 
+   * @return {Class} Class object
+   */
+  addClass(name, teacher) {
+    let newClass = new Class(name, teacher);
+    this.classes[name] = newClass;
```

Comment:
You can `return newClass` here so you can use it in your app.js file to include in your response object.